src/commands/InitCommand.js

```diff
@@ -54,6 +54,7 @@ export default class InitCommand extends Command {
 
   execute(callback) {
     this.ensurePackageJSON();
+    this.ensurePackagesDir();
```

Comment:
I'm actually calling it before so if it's not configured by the user no `packages` directory is made.